browser.js

```diff
@@ -1992,4 +1997,4 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 }());
-//# sourceMappingURL=browser.js.map
\ No newline at end of file
+//# sourceMappingURL=browser.js.map
```

Comment:
What's the story here? Was this automatically added by something? Is there a corresponding map that should be checked in?